cmd/applyconfig/applyconfig.go

```diff
@@ -545,6 +528,7 @@ func detectDryRunMethod(kubeconfig, context, username string) dryRunMethod {
 }
 
 func main() {
+	logrusutil.ComponentInit()
```

Comment:
Sorry, I should have said this before, too -- there is a subtle bug here in that `init()` fires first, so this overwrites that formatter.

IMHO we should not use `init()` or global `var`s here - it does not seem to have any utility over doing this in `main()`. A minimal fix would be to call `ComponentInit()` in `init()` and I can follow-up to stop using `init()` and globals.

Reply:
Oh, good, I missed that.  Updated.